persistence/persistence-api/src/main/java/org/cbioportal/persistence/ClinicalDataRepository.java

```diff
@@ -51,6 +51,10 @@ List<ClinicalData> fetchSampleClinicalDataClinicalTable(List<String> studyIds, L
                                                             Integer pageSize, Integer pageNumber, String searchTerm,
                                                             String sortBy, String direction);
```

Comment:
@BasLee Do you know why we don't cache the fetchSampleClinicalDataClinicalTable method? It looks like it's the only method doesn't been cached.

Reply:
no sorry, I was wondering myself

Reply:
Maybe it's because of the searchTerm parameter in this method? Not a big problem to leave it as is right now I guess